yala/setup.cfg

```diff
@@ -1,5 +1,13 @@
 [yala]
-isort args = --check
+isort args = --check --profile black
```

Comment:
It'll change the sample default settings config, since the [`--profile black`](https://pycqa.github.io/isort/docs/configuration/black_compatibility.html) is to ensure compatibility with `black` but I figured it would be the easiest approach to work out of the box with `isort`

Reply:
Indeed it can break existing non-black code. I reverted the change and added your suggestion eb4cb894aca84badf7953561643b5939e7d9c4f0